static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -515,7 +510,9 @@ const buildEpicCopy = (row: any, hasCountryName: boolean) => {
                 : heading,
         paragraphs:
             paragraphs && countryName
-                ? paragraphs.map(para => replaceCountryName(para, countryName))
+                ? paragraphs.map<string>(para =>
```

Comment:
I don't know why, but flow started complaining about `Missing type annotation` in the call to `map` here

Reply:
it reads oddly to me, but I can sympathize about flow complaining